### ISSUE TITLE
picard

### ISSUE THREAD

**Issue by chen5027** [ubccr/software-layer#198]
This is also a widely used one, which is essential for our daily analyses.  Could you install it so we can proceed in our research?  Thanks.  BTW, Dori helped me to verify that all the module requested here are not available yet on our HPC.

Dahong

**Comment by tonykew** [comment]
Test build of picard version 2.25.0 module complete - more info to follow

**Comment by tonykew** [comment]
picard version 2.25.0 has been published to the ccrsoft/2023.01 software release


```
vortex2$ module spider picard
-------------------------------------------------------------------------------------------------
  picard: picard/2.25.0-Java-11.0.16
-------------------------------------------------------------------------------------------------
    Description:
      A set of tools (in Java) for working with next generation sequencing data in the BAM format.

    Properties:
      Bioinformatic libraries/apps

    This module can be loaded directly: module load picard/2.25.0-Java-11.0.16
[...]
vortex2$ module load picard/2.25.0-Java-11.0.16
To execute picard run: java -jar $EBROOTPICARD/picard.jar
vortex2$ 
vortex2$ java -jar $EBROOTPICARD/picard.jar -h
USAGE: PicardCommandLine <program name> [-h]

Available Programs:
--------------------------------------------------------------------------------------
Base Calling:                                    Tools that process sequencing machine data, e.g. Illumina base calls, and detect sequencing level attributes, e.g. adapters
    CheckIlluminaDirectory                       Asserts the validity for specified Illumina basecalling data.  
    CollectIlluminaBasecallingMetrics            Collects Illumina Basecalling metrics for a sequencing run.  
[...]

```
